### ISSUE TITLE
{Linked:Bug50553332}[Adaptive Cards > Designer] - All Required fields (shown with a red Asterix) should be called out as required in narrator

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8931]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
URL: https://adaptivecards.io/designer/
OS Version: 24H2 (OS Build 26217.5000)
Browser Version: Version 127.0.2610.3 (Official build) dev (64-bit)
Screen Reader: Narrator

**Pre-Requisite:**
Run the below attached JSON file under 'Copy Card Payload editor.'

**Repro Steps:**
1. Open the above URL on the EDGE Dev browser.
2. Run the Json file.
3. Navigate to the Preview mode and press enter key to activate it.
4. Press tab key, navigate to all required fields, and observe the issue.

**Actual Result:**
Screen Reader does not announce required information while navigating on the edit field or combo box.

**Expected Result:**
Screen Reader should announce the required information with the edit and combo box.

**User Impact:**
It will impact the screen reader user as they will not be able to know which field is required due to this there might be a chance that they may skip the field.

**WCAG Reference Link:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/d08ae7ef-184e-4166-be54-32f0f98c2ba9

![Screen Reader is not announcing required information](https://github.com/microsoft/AdaptiveCards/assets/95913748/ff2f632e-e24e-4383-8335-91926e9d6bd7)

### Card JSON

```JSON
{ "type": "AdaptiveCard", "$schema": "http://adaptivecards.io/schemas/adaptive-card.json", "version": "1.5", "body": [ { "type": "ColumnSet", "spacing": "large", "columns": [ { "type": "Column", "width": "auto", "items": [ { "type": "Input.Text", "id": "NewEnvironmentName", "label": "${EnterNewEnvironmentTextBoxLabel}", "maxLength": 100, "isRequired": true, "errorMessage": "${DevBoxTextBoxErrorMessage}", "regex": "${DevBoxNameRegex}", "spacing": "large", "value": "${TextBoxText}" } ] } ] }, { "type": "ColumnSet", "spacing": "large", "columns": [ { "type": "Column", "width": "stretch", "items": [ { "type": "Input.ChoiceSet", "id": "ProjectsComboBox", "label": "${ProjectComboBoxLabel}", "devHomeChildChoiceSetId": "PoolsComboBox", "devHomeSelectionChangedDataForChildChoiceSet": "${SelectionChangedDataForChildChoiceSet}", "devHomeRefreshChildChoiceSetOnSelectionChanged": true, "isRequired": true, "errorMessage": "${DevBoxProjectComboBoxErrorMessage}", "style": "compact", "value": "${SelectedProjectIndex}", "choices": [ { "$data": "${ProjectList}", "title": "${title}", "value": "${value}" } ] } ] } ] }, { "type": "ColumnSet", "spacing": "large", "columns": [ { "type": "Column", "width": "stretch", "items": [ { "type": "Input.ChoiceSet", "id": "PoolsComboBox", "devHomeParentChoiceSetId": "ProjectsComboBox", "label": "${DevBoxPoolComboBoxLabel}", "isRequired": true, "errorMessage": "${DevBoxPoolsComboBoxErrorMessage}", "placeholder": "${DevBoxPoolsPlaceHolder}", "style": "compact", "value": "${SelectedPoolIndex}", "devHomeChoicesData": [ { "$data": "${PoolsList}", "title": "${title}", "subtitle": "${subtitle}", "value": "${value}" } ] } ] } ] }, { "type": "ActionSet", "id": "DevHomeTopLevelActionSet", "actions": [ { "id": "DevHomeMachineConfigurationNextButton", "type": "Action.Submit", "title": "${PrimaryButtonLabelForCreationFlow}" }, { "id": "DevHomeMachineConfigurationPreviousButton", "type": "Action.Submit", "title": "${SecondaryButtonLabelForCreationFlow}" } ] } ] }
```

**Comment by anna-dingler** [comment]
We have released another version of the website, and this fix is available. Thanks!

**Comment by vagpt** [comment]
Hi @anna-dingler 

Thanks for the updates.

Could you please confirm whether Dev team is still working on the Website as of now we are getting below screen while opening Designer tab, although for the other tab it is opening.

![Image](https://github.com/user-attachments/assets/43946029-c95c-403d-91a8-fd0a77c70513)

**Comment by anna-dingler** [comment]
Hi @vagpt, can you try clearing your cookies and refreshing? We are not seeing this on our side

**Comment by vagpt** [comment]
This issue is now working fine, hence closing this issue.

![Image](https://github.com/user-attachments/assets/88930dca-5150-465e-89cc-f7ddca6597f7)
